src/tree/AttachedServersTreeItem.ts

```diff
@@ -42,26 +51,44 @@ export class AttachedServersTreeItem implements IAzureParentTreeItem {
     }
 
     public async loadMoreChildren(_node: IAzureNode, clearCache: boolean): Promise<IAzureTreeItem[]> {
-        return this._attachedServers;
+        return this._attachedAccounts.length > 0 ? this._attachedAccounts : [{
+            contextValue: 'cosmosDBAttachDatabaseAccount',
+            label: 'Attach Database Account...',
+            id: 'cosmosDBAttachDatabaseAccount',
+            commandId: 'cosmosDB.attachDatabaseAccount'
+        }];
     }
 
-    public async attachNewServer(): Promise<void> {
-        const connectionString = await vscode.window.showInputBox({
-            placeHolder: 'mongodb://host:port',
-            prompt: 'Enter the connection string for your database account',
-            ignoreFocusOut: true
-        });
-
-        if (connectionString) {
-            const id = await this.getServerIdFromConnectionString(connectionString);
-            const node = new MongoAccountTreeItem(id, id, connectionString, 'mongoServer');
-            if (this._attachedServers.find(s => s.id === node.id)) {
-                vscode.window.showWarningMessage(`Mongo server '${node.id}' is already attached.`)
+    public async attachNewAccount(): Promise<void> {
```

Comment:
Cool!  I can attach my graph accounts using the DocDB API and see vertices and edges in the treeview!

Reply:
But update didn't work. :-)

Reply:
Haha yes I suppose you can do that - what error did you get? I successfully updated just now...